.pre-commit-hooks.yaml

```diff
@@ -35,3 +35,14 @@
   pass_filenames: false
   require_serial: true
   minimum_pre_commit_version: '2.8.0'
+- id: scala-cli-scalafmt-autofix
+  name: scalafmt format code via scala-cli
+  description: Automatically fix code formatting with scalafmt via scala-cli
+  entry: ./shell/scala-cli-scalafmt-autofix.sh
+  args: [--mode=changed]
```

Comment:
Maybe it's worth hardcoding the `--mode=changed` since `git diff --cached` will not work well with other modes.